File: Proyecto 10 - Material Branding/Certificados/print_pdfs.py
import asyncio
from playwright.async_api import async_playwright
import os

async def generate_pdfs():
    async with async_playwright() as p:
        browser = await p.chromium.launch(headless=True)
        page = await browser.new_page()
        
        desktop = r"C:\Users\ANGEL RAFAEL\Desktop"
        base_url = "http://localhost:8088"
        
        print("Generating PT PDF...")
        await page.goto(f"{base_url}/index.html", wait_until="networkidle")
        await page.emulate_media(media="print")
        await page.pdf(path=os.path.join(desktop, "Certificados_PT.pdf"), format="A4", landscape=True, print_background=True)
        
        print("Generating EN PDF...")
        await page.goto(f"{base_url}/index_en.html", wait_until="networkidle")
        await page.emulate_media(media="print")
        await page.pdf(path=os.path.join(desktop, "Certificados_EN.pdf"), format="A4", landscape=True, print_background=True)
        
        await browser.close()
        print("PDFs generated successfully.")

asyncio.run(generate_pdfs())

File: Proyecto 10 - Material Branding/Certificados/print_pdfs.js
const puppeteer = require('puppeteer');
const path = require('path');

(async () => {
    try {
        const browser = await puppeteer.launch();
        const page = await browser.newPage();
        
        const desktopPath = 'C:\\Users\\ANGEL RAFAEL\\Desktop';
        const baseUrl = 'file:///C:/Users/ANGEL%20RAFAEL/.gemini/antigravity/scratch/Certificados_Workshop';
        
        console.log('Generating PT PDF for Model 3...');
        await page.goto(`${baseUrl}/modelo3_pt.html`, { waitUntil: 'load' });
        await page.pdf({
            path: path.join(desktopPath, 'Certificado_Modelo3_PT.pdf'),
            format: 'A4',
            landscape: true,
            printBackground: true
        });
        
        console.log('Generating EN PDF for Model 3...');
        await page.goto(`${baseUrl}/modelo3_en.html`, { waitUntil: 'load' });
        await page.pdf({
            path: path.join(desktopPath, 'Certificado_Modelo3_EN.pdf'),
            format: 'A4',
            landscape: true,
            printBackground: true
        });
        
        await browser.close();
        console.log('PDFs generated successfully.');
    } catch (e) {
        console.error(e);
    }
})();
